crates/subspace-networking/src/node.rs

```diff
@@ -264,6 +265,8 @@ impl Node {
             .first()
             .ok_or(GetPiecesByRangeError::NoClosestPiecesFound)?;
 
+        trace!("Peer found {:?}, range: {:?} - {:?} ", peer_id, from, to);
```

Comment:
It is better to not use format most of the time:
```rust
trace!(found_peer = ?peer_id, range_from=?from, range_to=?to);
```

It would be helpful in future to search logs in smarter formats than txt. It will also let us introspect some values in runtime (for example in subspace-desktop)